### PR TITLE
Create Web Files

### PR DESCRIPTION
**Pull request by edgareu1**
A first draft of the web files.

Created multiple pages (*.html), a stylesheet (index.css) and a script (index.js).

Pages were created for:
1. Content display (index.html);
2. Error display (error-pages/*.html);
3. Test redirects (redirects/*.html);
4. Test post/delete requests (form*.html).

A cgi folder was added (cgi-bin) to host our cgi scripts. Empty for now.

While the server is not working, `http-server` is a great tool to test this pages visually (does not work for CGI testing).

### CODE REVIEW

**Review by PukieDiederik — CHANGES_REQUESTED** [review]
Good start but i'd like to see the following changes:
- Delete/Justify redirect pages
- Make href use relative file paths instead of absolute. This is because if you have this website on `[authority]/test` for example `/index.js` looks for `[authority]/index.js` instead of `[authority]/test/index.js`

File: var/www/files/.keep
[the remark this answers is not in the document]
Comment (edgareu1) [reply var/www/files/.keep:1]: To save the uploaded files

File: var/www/index.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+						</li>
+
+						<li>
+							<a href="/form-delete">
+								Form Delete
+							</a>
+						</li>
+
+						<li>
+							<a href="/redirects/permanent">
```
[the remark this answers is not in the document]
Comment (edgareu1) [reply var/www/index.html:53]: Just removed it the `var/www/redirect` folder

File: var/www/error-pages/403.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>42 Webserver - Error 403</title>
+
+		<link rel="stylesheet" href="/index.css">
```
[the remark this answers is not in the document]
Comment (edgareu1) [reply var/www/error-pages/403.html]: Done

File: var/www/index.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+						Say Hello
+					</button>
+				</section>
+
+				<section>
+					<h2 class="subtitle">Website map:</h2>
+
+					<ul>
+						<li>
+							<a href="/form">
```
[the remark this answers is not in the document]
Comment (edgareu1) [reply var/www/index.html]: Done

**Review by PukieDiederik — APPROVED** [review]
Everything looks good to me, really good job! For the next steps I recommend either creating a config file (though this might be better to wait after the parsing is implemented), or implementing the CGI files.

**Pull request merged**